rpcd/playbooks/roles/rpc_support/tasks/support_preinstall.yml

```diff
@@ -33,3 +33,20 @@
   when: inventory_hostname in groups['hosts']
   tags:
     - support_packages
+
+- name: Remove host packages from containers
+  apt:
+    pkg: "{{ item }}"
+    state: absent
+    update_cache: yes
+    cache_valid_time: 600
+  with_items:
+    - debbootstrap
```

Comment:
The package name should be 'debootstrap'. This package seems to be a requirement for the repo containers.